plugins/com.google.cloud.tools.eclipse.appengine.newproject/build.properties

```diff
@@ -3,6 +3,7 @@ output.. = target/classes/
 bin.includes = META-INF/,\
                .,\
                plugin.properties,\
-               plugin.xml
+               plugin.xml,\
+               help-contexts/
```

Comment:
Eclipse plugins typically put this file at the root and call it `helpContexts.xml`.